.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,72 @@
+name: Run Integration Tests for EKS Addon
+env:
+  TERRAFORM_AWS_ASSUME_ROLE_DURATION: 14400 # 4 hours
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
```

Comment:
probably ignore any workflow changes except for this test:

```
- '.github/**'
- '!.github/workflows/integration-test.yml'
```

Reply:
Oh yeah, will add it.

Reply:
https://github.com/aws/amazon-cloudwatch-agent-operator/actions/runs/6122765468 is successful integration test ran